ibm/mas_devops/roles/ocp_provision/tasks/providers/fyre.yml

```diff
@@ -39,7 +40,7 @@
 # -----------------------------------------------------------------------------
 - name: "fyre : Check if cluster already exists"
   uri:
-    url: "https://ocpapi.svl.ibm.com/v1/check_hostname/{{ cluster_name }}"
+    url: "https://ocpapi.svl.ibm.com/v1/check_hostname/{{ cluster_name }}?site={{ fyre_site }}"
```

Comment:
Check hostname doesn't need it.

Reply:
Same as:

<img width="1004" alt="image" src="https://github.com/ibm-mas/ansible-devops/assets/15021471/bcb96c08-8333-47ad-aba7-7c59772ee38f">